tests/integration/src/test/scala/mesosphere/marathon/integration/SequentialIntegrationTests.scala

```diff
@@ -0,0 +1,12 @@
+package mesosphere.marathon
+package integration
+
+import org.scalatest.Sequential
+
+/**
+  * Wrapper that prohibits parallel test run of the respective test suites.
+  */
+class SequentialIntegrationTests extends Sequential(
```

Comment:
Why is this required? I'm not a big face of reintroducing it.

Reply:
It's a temporary solution to make test that interfering each other pass. We should tackle this in the follow-up.